--- a/tk/tk.tcl	2008-06-12 18:01:56.000000000 +0300
+++ b/tk/tk.tcl	2014-02-14 20:57:48.814356875 +0200
@@ -406,15 +406,7 @@
         namespace eval :: [list source [file join $::tk_library $file.tcl]]
     }
     namespace eval ::tk {
-	SourceLibFile button
-	SourceLibFile entry
 	SourceLibFile listbox
-	SourceLibFile menu
-	SourceLibFile panedwindow
-	SourceLibFile scale
-	SourceLibFile scrlbar
-	SourceLibFile spinbox
-	SourceLibFile text
     }
 }
 # ----------------------------------------------------------------------
