--- a/tk/ttk/ttk.tcl	2008-06-12 18:01:56.000000000 +0300
+++ b/tk/ttk/ttk.tcl	2014-05-27 12:45:57.497286576 +0300
@@ -14,8 +14,6 @@
     }
 }
 
-source [file join $::ttk::library fonts.tcl]
-source [file join $::ttk::library cursors.tcl]
 source [file join $::ttk::library utils.tcl]
 
 ## ttk::deprecated $old $new --
@@ -97,16 +95,9 @@
 ### Load widget bindings.
 #
 source [file join $::ttk::library button.tcl]
-source [file join $::ttk::library menubutton.tcl]
 source [file join $::ttk::library scrollbar.tcl]
-source [file join $::ttk::library scale.tcl]
-source [file join $::ttk::library progress.tcl]
-source [file join $::ttk::library notebook.tcl]
-source [file join $::ttk::library panedwindow.tcl]
 source [file join $::ttk::library entry.tcl]
 source [file join $::ttk::library combobox.tcl]	;# dependency: entry.tcl
-source [file join $::ttk::library treeview.tcl]
-source [file join $::ttk::library sizegrip.tcl]
 
 ## Label and Labelframe bindings:
 #  (not enough to justify their own file...)
@@ -119,17 +110,10 @@
 proc ttk::LoadThemes {} {
     variable library
 
-    # "default" always present:
-    uplevel #0 [list source [file join $library defaults.tcl]] 
-
     set builtinThemes [style theme names]
     foreach {theme script} {
-	classic 	classicTheme.tcl
-	alt 		altTheme.tcl
-	clam 		clamTheme.tcl
 	winnative	winTheme.tcl
 	xpnative	xpTheme.tcl
-	aqua 		aquaTheme.tcl
     } {
 	if {[lsearch -exact $builtinThemes $theme] >= 0} {
 	    uplevel #0 [list source [file join $library $script]]
